content/python/concepts/strings/strings.md

```diff
@@ -101,7 +101,16 @@ print(my_string)
 # Output: It's a lovely day!
 ```
 
-Python also has a series of non-printable characters that can modify strings. For example, `\n` will add a new line, and `\t` will add a tab:
+You could also get around this problem by wrapping strings containing `'` characters in double quotes:
+
+```py
+my_string = "It's a lovely day!"
+
+print(my_string)
+# Output: It's a lovely day!
+```
+
+Python also has a series of non-printing characters that can modify strings. For example, `\n` adds a new line, and `\t` adds a tab:
```

Comment:
```suggestion
Python also has a series of non-printing characters that can modify strings. For example, `\n` adds a new line and `\t` adds a tab:
```

A comma before "and" is usually applied when there are >2 listed items/ideas. I see the same thing on line 123 https://www.grammarly.com/blog/comma-before-and/

Reply:
Yes; this was my love of the Oxford comma coming through; fine to have it without.